go.mod

```diff
@@ -1,9 +1,11 @@
-module github.com/gig-tech/ovc-sdk-go/v2
+module github.com/gig-tech/ovc-sdk-go/v3
 
 go 1.11
 
 require (
 	github.com/dgrijalva/jwt-go v3.2.0+incompatible
+	github.com/gig-tech/ovc-sdk-go v1.4.1 // indirect
```

Comment:
Is that intentional?

Reply:
That was added by Go
After a `go mod tidy`it got removed